pathd/path_zebra.c

```diff
@@ -390,6 +390,75 @@ void path_nht_removed(struct srte_candidate *candidate)
 			  candidate->preference);
 }
 
+/**
+ * Send SRv6 SID to ZEBRA for installation or deletion.
+ *
+ * @param cmd		ZEBRA_ROUTE_ADD or ZEBRA_ROUTE_DELETE
+ * @param sid		SRv6 BSID to install or delete
+ * @param prefixlen	Prefix length
+ * @param oif		Outgoing interface
+ * @param action	SID action
+ * @param context	SID context
+ */
+void path_zebra_send_bsid(const struct in6_addr *bsid, ifindex_t oif,
+			  enum seg6local_action_t action,
+			  struct in6_addr *srv6_segs, int num_segs)
+{
+	struct prefix_ipv6 p = {};
+	struct zapi_route api = {};
+	struct zapi_nexthop *znh;
+	uint16_t prefixlen = IPV6_MAX_BITLEN;
+	char opaque[1024];
+
+	if (prefixlen > IPV6_MAX_BITLEN) {
+		flog_warn(EC_LIB_DEVELOPMENT, "%s: wrong prefixlen %u",
+			  __func__, prefixlen);
+		return;
+	}
+
+	opaque[0] = '\0';
+	p.family = AF_INET6;
+	p.prefixlen = prefixlen;
+	memcpy(&p.prefix, bsid, IPV6_MAX_BYTELEN);
+
+	api.vrf_id = VRF_DEFAULT;
+	api.type = ZEBRA_ROUTE_SRTE;
+	api.distance = 50;
+
```

Comment:
use the ZEBRA_SHARP_DISTANCE_DEFAULT define value.
if the value has to be changed, then this will be in zebra.h that it will be done.